### PR TITLE
Improve output formatting

### LINKED ISSUE #20: Output formatting improvements
Currently the output doesn't produce nice human-readable descriptions, and doesn't differentiate between singular and plural values.

```
❯ target/release/cpc '123 kph'
123 KilometersPerHour

❯ target/release/cpc '1 kph'
1 KilometersPerHour
```

It would be good if the output of the above was `123 kilometres per hour` and `1 kilometre per hour`.

### PR DESCRIPTION
**Pull request by FinnRG**
Depends on #23 (Merge conflicts).
Closes #20.

### CODE REVIEW

**Comment by probablykasper** [comment]
Would be great if it could be done like this, but unfortunately it doesn't work for all units. The first that comes to mind, `2 km/h` becomes `2 Kilometers per hours`.

I think the best solution is sadly to write them all out manually :/

**Comment by djmattyg007** [comment]
I have to agree. At a glance it also looks like `0` would be formatted as `0 kilometer`, rather than `0 kilometers`.

File: src/lib.rs [hunk 1]
```diff
@@ -69,7 +69,13 @@ impl Display for Number {
     let fixed_value = self.value + d128!(0);
     let output = match self.unit {
       Unit::NoUnit => format!("{}", fixed_value),
-      unit => format!("{} {:?}", fixed_value, unit),
+      unit => {
+        if self.value > d128!(1) {
+          format!("{} {}s", self.value, unit.to_string())
```
Comment (Octalbyte) [inline src/lib.rs:74]: Maybe instead of ` >` we should use `!=`

Comment (Octalbyte) [reply src/lib.rs:74]: This would solve the issue with 0.

Comment (kasper9n) [reply src/lib.rs:74]: Either way you would still end up with "kilometers per hours"

**Pull request closed without merging**